### PR TITLE
Assistant V2: Post agent route

### PR DESCRIPTION
**Pull request by PopDaph**
Adding a route to create a new Assistant: `POST /api/w/[wId]/assistants assistant creation`.
Part of: https://github.com/dust-tt/dust/issues/1347

Not tested!

### CODE REVIEW

File: front/lib/api/assistant/configuration.ts [hunk 1]
```diff
@@ -196,7 +196,7 @@ export async function createAgentConfiguration(
     action,
   }: {
     name: string;
-    pictureUrl: string;
+    pictureUrl: string | null;
```
Comment (spolu) [inline front/lib/api/assistant/configuration.ts:199]: We don't want null do we?

File: front/pages/api/w/[wId]/assistants/index.ts [hunk 2]
```diff
@@ -10,10 +17,79 @@ import { AgentConfigurationType } from "@app/types/assistant/agent";
 export type GetAssistantResponseBody = {
   assistants: AgentConfigurationType[];
 };
+export type PostAssistantResponseBody = {
+  assistant: AgentConfigurationType;
+};
+
+const PostAssistantResponseBodySchema = t.type({
```
Comment (spolu) [inline front/pages/api/w/[wId]/assistants/index.ts:24]: :fire:

File: front/pages/api/w/[wId]/assistants/index.ts [hunk 3]
```diff
@@ -10,10 +17,79 @@ import { AgentConfigurationType } from "@app/types/assistant/agent";
...
+  assistant: t.type({
+    name: t.string,
+    pictureUrl: t.union([t.string, t.null]),
```
Comment (spolu) [inline front/pages/api/w/[wId]/assistants/index.ts:27]: why null?

**Review by spolu — COMMENTED** [review]
Looks great question on pictureUrl?

File: front/lib/api/assistant/configuration.ts (hunk 1 above)
Comment (PopDaph) [reply front/lib/api/assistant/configuration.ts]: Yes, you're right, I updated!

File: front/pages/api/w/[wId]/assistants/index.ts (hunk 3 above)
Comment (PopDaph) [reply front/pages/api/w/[wId]/assistants/index.ts]: Removed!

**Review by spolu — APPROVED** [review]

**Pull request merged**